docs/scripting/resources/animations.md

```diff
@@ -0,0 +1,4015 @@
+---
+id: animations
+title: Animations
+description: A list of all Animations
+tags: []
+---
+
+**Notes**
```

Comment:
This should probably be a note admonition https://v2.docusaurus.io/docs/markdown-features/#calloutsadmonitions

Reply:
i've just readed it ok sec

Reply:
Done.